frontend/src/components/CensusDataChart/CensusDataChart.tsx

```diff
@@ -28,19 +29,48 @@ export let CensusDataChart: React.StatelessComponent<Props> = ({ percent, measur
     let options = {
         scales: {
             xAxes: [{
-                scaleLabel: { display: true, labelString: xLabel }
+                scaleLabel: { display: true, labelString: xLabel },
+                ticks: {
+                    callback: (value: any) => {
+                        return `${formatNumber(value)}`
+                     }
+                }
             }]
+        },
+        tooltips: {
+            callbacks: {
+                label: (tooltipItem: ChartTooltipItem, data: any) => {
+                    let label = data.datasets[tooltipItem.datasetIndex].label
+                    return `${label}: ${formatNumber(tooltipItem.xLabel)}`
+                }
+            }
         }
     }
 
     let percentOptions = {
             scales: {
                 yAxes: [{
                     stacked: true
+                }],
+                xAxes: [{
+                    ticks: {
+                        callback: (value: any) => {
+                            return `${formatPercentage(value, 0)}`
+                         }
+                    }
                 }]
             },
             plugins: {
                 stacked100: { enable: true }
+            },
+            tooltips: {
+                callbacks: {
```

Comment:
@ericboucher wondering if I should pull out the callbacks into functions and be a bit smarter about ternary operators

Reply:
Why not, I think the doughnut was using something like that if it helps.